library-best-practices/JLBP-4.md

```diff
@@ -0,0 +1,31 @@
+[JLBP-4] Avoid dependencies on unstable libraries and features
+--------------------------------------------------------------
+
+- Unstable libraries are libraries that allow for breaking changes to their
+  Public API within the same major version. For libraries following semver, this
+  means libraries with a 0.x.y version. (See [JLBP-3](JLBP-3.md) for more
+  details on the recommendations for following semver.)
+- Unstable features are features that are not part of the stable Public API of a
+  stable library, often marked with annotations like `@Beta`.  (See
+  [JLBP-3](JLBP-3.md) for more details on the recommendations for annotating
+  unstable features.)
+- If your library depends on an unstable library or feature, and that feature
+  experiences a breaking change between versions, your library will be locked to
+  a specific version of your dependency.
```

Comment:
your dependency --> the dependency

Reply:
I don't feel like that reads quite right...